src/west/commands/project.py

```diff
@@ -588,11 +617,10 @@ def _projects(args, listed_must_be_cloned=True, include_meta=False):
         uncloned = [prj.name for prj in res
                     if not _cloned(prj) and prj.name not in META_NAMES]
         if uncloned:
-            log.die('Uncloned project{}: {}.'.
-                    format('s' if len(uncloned) > 1 else '',
-                           ", ".join(uncloned)),
-                    'Please run "west clone {}"'.format(' '.join(uncloned)),
-                    'to clone.')
+            log.die('The following projects are not cloned: {}. Please clone '
+                    "them first with 'west clone' (or with 'west fetch', "
+                    "if you don't want an initial topic branch created)."
```

Comment:
Adding additional text does not solve the problem, which is, again:

1. nobody we have ever asked wants a detached head and "west fetch" for an uncloned project would create one
2. making people decide whether they want a detached head or not when we know they don't is wasting their time

Reply:
k

Changed to get things moving.